docs/src/prototypes/ComponentArchitecture/BasePrefixedInput.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
```

Comment:
Are we not using `;` at the end of statements now?

Reply:
No. Most all of this repo is configured via prettier to omit semicolons.